change/@fluentui-react-message-bar-dcc6ade3-1e4b-44ab-b29b-14b1cafc1320.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "fix(react-message-bar): Use useAnnounce() instead of the deprecated useAnnounce_unstable()",
```

```suggestion
  "comment": "chore(react-message-bar): Use useAnnounce() instead of the deprecated useAnnounce_unstable()",
```

tiny tiny nit, not really actually important